### DISCUSSION TITLE
设置目录显示级别（目录显示深度）后，出现格式不匹配

### DISCUSSION THREAD

**Discussion by Auyuir** [tuna/thuthesis#853] [Q&A] [answered]
在顶层文件thuthesis-example.tex 的 \begin{document}语句之前使用语句

```\setcounter{tocdepth}{4}```

，试图把目录显示级别从3改成4。

但是出现了不美观的格式：
![image](https://user-images.githubusercontent.com/31274112/230535335-027679c4-ed78-4825-96d2-ef315e25276e.png)

请问：
1、研究生院是否有要求目录只显示到3级标题，不能显示4级标题？
2、在本模板中这样使用setcounter{tocdepth}是否正确？

**Comment by zepinglee** [comment, marked as answer]
自己翻一下《写作指南》。

<img width="613" alt="Screenshot 2023-04-07 at 12 50 40" src="https://user-images.githubusercontent.com/12290822/230543124-9a3339ff-e82f-49d4-b0d8-bb11333dac26.png">

**Reply by Auyuir** [reply]
谢谢！
